projectfiles/.gitignore

```diff
@@ -0,0 +1,3 @@
+/Xcode-iOS
+/*.xcworkspace
+*.xcworkspacedata
```

Comment:
I'm not sure how you get a 1GB project file. o_o On master, it's just the pbxproj file and one of the xcscheme files that are actually committed within the XCode project file package; the rest is ignored by the version control. Surely those sources and binaries aren't actually encoded into the main pbxproj project file?